From 7143b747e2fd0b7702228fcf9de6a7dee3195216 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Mon, 1 Jan 2018 17:04:31 +0200
Subject: [PATCH 159/249] lineage: Enable prebuilts/lineage-sdk

* Empty for now, to be filled with Lineage SDK jars

Change-Id: Ic2d2496b6e2bc6cf408b52cc641ee7fcf4e21666
---
 snippets/lineage.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index ebfd146..6b120df 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -50,7 +50,7 @@
   <!--<project path="packages/apps/Updater" name="LineageOS/android_packages_apps_Updater" />-->
   <!--<project path="packages/providers/DataUsageProvider" name="LineageOS/android_packages_providers_DataUsageProvider" />-->
   <project path="packages/providers/WeatherProvider" name="LineageOS/android_packages_providers_WeatherProvider" />
-  <!--<project path="prebuilts/lineage-sdk" name="LineageOS/android_prebuilts_lineage-sdk" clone-depth="1" />-->
+  <project path="prebuilts/lineage-sdk" name="LineageOS/android_prebuilts_lineage-sdk" clone-depth="1" />
   <!--<project path="system/extras/su" name="LineageOS/android_system_extras_su" />-->
   <project path="vendor/lineage" name="LineageOS/android_vendor_lineage" />
 
-- 
2.17.1

